Source/r_plane.c

```diff
@@ -375,6 +379,9 @@ static void do_draw_plane(visplane_t *pl)
 
 	    // Vertical offset allows careful sky positioning.
 
+	    if (dc_texheight >= 200)
```

Comment:
I'd say, if we decide to second-guess `dc_texturemid` for tall skies, we should completely ignore `s->rowoffset` and just let it start at the top, as we do for regular sky textures.